.github/workflows/cd_push.yml

```diff
@@ -44,7 +48,7 @@ jobs:
         uses: rtCamp/action-slack-notify@v2
         if: failure() && (github.ref_name  == 'development' || github.ref_name  == 'master' || github.ref_name  == 'production')
         env:
-          SLACK_CHANNEL: taas-dev
+          SLACK_CHANNEL: taas-github
```

Comment:
I think failed deploys should be kept in #taas-dev 

Github channel is spammed allot by github stuff and this would easily drown